tests/test_parser.py

```diff
@@ -566,6 +577,24 @@ def test_variables(self):
         assert ('x2' in variables and
                 variables['x2'] == (88342528, Type('int')))
 
+        # Test int div 9 / 2 should be 4
+        assert ('x3' in variables and
```

Comment:
```suggestion
        assert (
```

the indexing will anyway if the key does not exist in the dict. No need to test it separately.